### DISCUSSION TITLE
I have not received your education benefits within 5 days of your approval status

### DISCUSSION THREAD

**Discussion by nkhokhla** [community/community#139841] [GitHub Education]
### Hi everyone,

Hello GitHub Education Team,

I hope you’re doing well!

I was approved for GitHub Education benefits, and my academic affiliation was verified more than 5 days ago. However, I still haven’t received my education benefits. I understand there might be some delays during busy periods, but I wanted to raise this here just in case. 

Could you kindly assist in resolving this? Let me know if you need any additional information from my end.

Thank you for your support!

**Comment by davevad93** [comment]
Hi @nkhokhla ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @nkhokhla was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @nkhokhla was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "All offers" section and select one of the available offers, if you can redeem them you are fine, if you can't I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268
